api/marketplace/v1/marketplace.proto

```diff
@@ -193,6 +195,23 @@ message News {
   repeated Action actions = 5;
 }
 
+message DAppGroup {
+  string id = 1;
+  string groupName = 2;
+  string icon = 3;
+  repeated string options = 4;
+}
+
+message DApp {
+  string id = 1;
+  string title = 2;
+  string description = 3;
+  string icon = 4;
+  string route = 5;
+  string groupKey = 6;
+  string airtableId = 7;
```

Comment:
do you really need this?
I would prefer that we don't have any airtable specific things in this interface so there won't be problems if we stop using airtable as data source

Reply:
I **really** need this

Reply:
yes, is needed.
I can change the name to `linking ID` if you want.
This will better convey it's intent.